From 4ab673e77653d2342556c1c7495cbeac59887557 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Diego=20Elio=20'Flameeyes'=20Petten=C3=B2?= <flameeyes@gmail.com>
Date: Sun, 14 Feb 2010 21:22:30 +0100
Subject: [PATCH] Don't force Hanna usage for documentation building.

Since Hanna is not always available (and requires a specific version of
RDoc gem), don't force to use it with no alternative. Instead, if not
available, fallback to the standard rdoc support within rake.
---
 Rakefile |    7 ++++++-
 1 files changed, 6 insertions(+), 1 deletions(-)

diff --git a/Rakefile b/Rakefile
index 85d1c50..d1a2721 100644
--- a/Rakefile
+++ b/Rakefile
@@ -1,7 +1,6 @@
 require 'rubygems'
 require 'rake/clean'
 require 'rake/gempackagetask'
-require 'hanna/rdoctask'
 require 'fileutils'
 include FileUtils
  
@@ -66,6 +65,12 @@ end
 
 # RUBY DOCS TASK ==================================
 
+begin
+  require 'hanna/rdoctask'
+rescue LoadError
+  require 'rake/rdoctask'
+end
+
 Rake::RDocTask.new do |t|
 	t.rdoc_dir = 'doc'
 	t.title    = @spec.summary
-- 
1.7.0

